recipes-app/iot2050-firmware-update/files/iot2050-firmware-update.tmpl

```diff
@@ -451,6 +457,7 @@ class ManagedFirmwareUpdate(FirmwareUpdate):
     '''
     _update_conf_json = 'update.conf.json'
     uboot_default_env_file_name = 'u-boot-initial-env'
+    firmware_names = ''
```

Comment:
This is in class space? although currently we only support singleton for updater...